Jenkinsfile

```diff
@@ -12,11 +12,6 @@ pipeline {
                     // Clean up any unwanted files lying about after the previous build.
                     sh "git clean -fxd --exclude='.venv'"
```

Comment:
The tests stages (i.e. pytest and behave) are missing in the Jenkins file at https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/454

Reply:
Also, there should be a Tox stage within the devtools project's Jenkins file isnt it?